src/components/Accordion/Accordion.spec.tsx

```diff
@@ -66,6 +66,36 @@ describe("Accordion Component", () => {
         cy.get("@onClickStub").should("be.calledOnce");
     });
 
+    it("opens select sections initially", () => {
+        mount(
+            <Accordion>
+                <AccordionItem header={{ children: "1", active: true }}>
+                    <TextInput />
+                </AccordionItem>
+                <AccordionItem header={{ children: "2" }}>
+                    <TextInput />
+                </AccordionItem>
+                <AccordionItem header={{ children: "3" }}>3</AccordionItem>
```

Comment:
You can add multiple tests
- It should have none opened by default
- It should have one opened
- It should have one opened which is not the first one
- It should have multiple opened